### ISSUE TITLE
Bridge not built/tested on OSX anymore

### ISSUE THREAD

**Issue by mikaelarguedas** [ros2/build_farmer#116]
The packaging job don't seem to test the `ros1_bridge` on any node anymore.

CI expects ROS 1 to be installed at : `/Users/osrf/kinetic/install_isolated`
But as it's not there it prints `WW> Asked to use ROS 1 but the setup file was not found at '/Users/osrf/kinetic/install_isolated/setup.sh'` instead

My guess is that only Dosa had ROS1 built and installed at that location

**Comment by dhood** [comment]
For reference I don't remember this being tested on OS X for the last 2 years (just to give some sort of bound on when this behaviour changed)

**Comment by mikaelarguedas** [comment]
Yeah from looking into it a few days ago, there are builds from a while back where it was built but not tested. As these jobs were on Dosa, my guess is that Dosa was the only machine with ROS 1 installed. It may have stopped being built the day we switched to `kinetic` (and did not installed kinetic on Dosa). Or maybe only when we decommissioned Dosa, I didnt look thoroughly through the packaging job history to find out as I think either way we should reenable it (and to update the MacOS node installation instructions accordingly). The README of the bridge claims to work on MacOS but we don't build test or ship it in our fat archives.
